docs/ltr-features.md

```diff
@@ -0,0 +1,90 @@
+# Features
+| Feature No.     | Description                                     |
+|-----------------|-------------------------------------------------|
+|      1          | [IBM Model1 in title\_unlemm](../src/main/java/io/anserini/ltr/feature/IBMModel1.java)            
```

Comment:
Based on my understanding, we specify computing the features on _which field_ separately, right?

So I think the table should only contain the features... and then in the Pyserini docs we say, we compute the scores over _x, y, z_ fields?

Also, let's _not_ number the features, because if we add, we'll need to renumber. Give the features names - similar to Java Class is fine, but let's refer to them uniquely.

Reply:
I use DFR_GL2 convention not jave consistent way in the doc, since I think this is easier for people to read? Or do you think we want DfrGl2 in doc as well?

Reply:
In the table, `DFR_GL2` is good.